sites/public/src/pages/sign-in.tsx

```diff
@@ -119,7 +123,8 @@ const SignIn = () => {
         undefined,
         undefined,
         undefined,
-        reCaptchaEnabled ? reCaptchaToken : undefined
+        reCaptchaEnabled ? reCaptchaToken : undefined,
+        !notChecked ? true : undefined
```

Comment:
Can this just be `!notChecked`?

Reply:
I didn't want to send back a false in the case where the user has already accepted the terms so `notChecked` is FALSE